clients/web/src/stylesheets/widgets/metadataWidget.styl

```diff
@@ -62,11 +62,11 @@
     border-bottom 1px solid #f0f0f0
 
     button
```

Comment:
None of these styles have any effect, since the classes are now named `.jsoneditor-menu`, `.jsoneditor-search`, etc. Furthermore, if you correct the class names, these styles actually cause the appearance to break.

Also, the `border` setting of `.jsoneditor` is being overridden by the more-specific upstream style, which operates on `div.jsoneditor`. If you want this border style to have any effect, you'll need to have an `!important`.

Reply:
Supplemental PR addressing these concerns: #2359